app/src/main/res/values-de/strings.xml

```diff
@@ -8,13 +8,13 @@
 
     <!-- ==============  Edit and add a plant  ============== -->
     <!-- Edit and add a plant: Activity title -->
-    <string name="title_edit_plant">Pflanze Bearbeiten</string>
```

Comment:
Did this:

```shell
~/mundraub-android$ tx push --translations --language de --resource mundraub-android-app.strings-xml
tx INFO: Pushing resource mundraub-android-app.strings-xml
tx WARNING: Pushing 'de' translations (file: app/src/main/res/values-de/strings.xml)
tx INFO: Done.
```
![image](https://user-images.githubusercontent.com/564768/65353232-e8711280-dbec-11e9-8670-2bf7a9a097ae.png)

It worked. It is pushed.